### ISSUE TITLE
Does lxd-webgui support both local & remote lxd containers

### ISSUE THREAD

**Issue by bmullan** [dobin/lxd-webgui#35]
The ReadMe doesn't say but does lxd-webgui support both local server LXD containers as well as remote LXD containers via the REST API like the LXD cli does ?

thanks
brian

**Comment by dobin** [comment]
Yes you can connect to remote LXD server instances. I'll update the readme accordingly.
